course_content/notebooks/iris_intro.ipynb

```diff
@@ -757,7 +757,7 @@
    "source": [
     "### Time Constraints\n",
     "\n",
-    "It is common to want to build a constraint for time. This can be achieved by comparing cells containing datetimes. The functionality can be enabled globally within the session (and will be enabled by default in v2.0, the next release of Iris) with:"
+    "It is common to want to build a constraint for time. This can be achieved by comparing cells containing datetimes, which is available by default in Iris v2.0. If you are using Iris v1.x, please uncomment and run the line below to enable this functionality globally within the session:"
```

Comment:
"line" --> "code cell"

Maybe also remove the point versions from the Iris versions? (That is, v2.0 --> v2; v1.x --> v1) Or possibly, just v2.0 --> v2.x. But I'm not sold on this idea, so feel free to leave out this second bit.